code/game/mecha/mecha.dm

```diff
@@ -840,6 +840,9 @@
 	if(repairing)
 		to_chat(user, "<span class='notice'>[src] is currently being repaired!</span>")
 		return
+	if(state == 0)
```

Comment:
Ah yes state... It'll be out of scope to refactor this into defines but can you place a comment saying what the state means? Makes it easier for later

Reply:
Done!